apis/refs/v1beta1/computerefs.go

```diff
@@ -27,24 +26,15 @@ import (
 )
 
 type ComputeNetworkRef struct {
-	/* The compute network selflink of form "projects/<project>/global/networks/<network>", when not managed by KCC. */
+	/* An external value of a `ComputeNetwork` resource, when not managed by KCC. */
```

Comment:
I may not be the most up to date with the pain points here but I somewhat like the self documenting aspect of `projects/<project>/global/networks/<network>`. As a user, I see it in the CRD and know what the expected format is.

Regarding validation, I think about how most validation that GCP makes we probably want to shift left so that we don't make extra API calls. But I'll let others chime in here if it doesn't apply.

Reply:
Updating the comment because I feel External can be any format today, there should be no "expected" format for external. But we can have a "suggested" format for user, based on our understanding about the API.